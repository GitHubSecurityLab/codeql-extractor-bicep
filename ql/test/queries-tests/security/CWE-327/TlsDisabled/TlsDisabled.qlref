security/CWE-327/TlsDisabled.ql