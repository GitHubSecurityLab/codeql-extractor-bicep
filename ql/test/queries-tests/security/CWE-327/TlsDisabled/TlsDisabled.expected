| app.bicep:22:1:29:1 | RedisCacheResource | TLS is disabled for this resource |
